config_app/init/certs_create.sh

```diff
@@ -14,9 +14,15 @@ else
     # Create certs to secure connections while uploading config for secrets
     # echo '{"CN":"CA","key":{"algo":"rsa","size":2048}}' | cfssl gencert -initca - | cfssljson -bare quay-config
     mkdir -p /certificates; cd /certificates
-    openssl req -new -newkey rsa:4096 -days 3650 -nodes -x509 \
-        -subj "/C=US/ST=NY/L=NYC/O=Dis/CN=self-signed" \
-        -keyout quay-config-key.pem  -out quay-config.pem
+    if [ -z "$QUAY_CONFIG_HOSTNAME" ];      then
+        openssl req -new -newkey rsa:4096 -days 3650 -nodes -x509 \
+            -subj "/C=US/ST=NY/L=NYC/O=Dis/CN=self-signed" \
+            -keyout quay-config-key.pem  -out quay-config.pem
+        else
+            openssl req -new -newkey rsa:4096 -days 3650 -nodes -x509 \
+            -subj "/C=US/ST=NY/L=NYC/O=Dis/CN=${QUAY_CONFIG_HOSTNAME}" \
+            -keyout quay-config-key.pem  -out quay-config.pem
+    fi
```

Comment:
indentation doesn't look correct

Reply:
Corrected to be the same as before.

Reply:
I force pushed changes so it should be okay now.